src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcTransport.cs

```diff
@@ -221,10 +221,13 @@ internal class PidIpcEndpoint : IpcEndpoint
     {
         public static string IpcRootPath { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? @"\\.\pipe\" : Path.GetTempPath();
         public static string DiagnosticsPortPattern { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? @"^dotnet-diagnostic-(\d+)$" : @"^dotnet-diagnostic-(\d+)-(\d+)-socket$";
-
+        // Format strings as private const members
+        private const string _defaultAddressFormatWindows = "dotnet-diagnostic-{0}";
+        private const string _dsrouterAddressFormatWindows = "dotnet-diagnostic-dsrouter-{0}";
+        private const string _defaultAddressFormatNonWindows = "dotnet-diagnostic-{0}-{1}-socket";
+        private const string _dsrouterAddressFormatNonWindows = "dotnet-diagnostic-dsrouter-{0}-*-socket";
```

Comment:
We should use consistent format with default, so * should be part of the formatting and not the format string.